scripts/codecov.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash
+FILENAME=codecov
+curl -s https://codecov.io/bash > $FILENAME
+CODECOV_VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
+VALIDATION_FAILURE=0
+for i in 1 256 512
+do
+  IS_DIFF=$(diff <(shasum -a $i $FILENAME) <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/$CODECOV_VERSION/SHA${i}SUM))
```

Comment:
Are you sure you have shasum in your environment? I am more familiar with sha1sum or sha256sum in Linux environments.

Reply:
It seems so. That command is included in the image picked from the CI